### ISSUE TITLE
macOS не подключается Tableau

### ISSUE THREAD

**Issue by sablastik** [ClickHouse/clickhouse-odbc#319]
Сервер 20.6.6.7
Драйвер 1.1.8.20200710

Строка подключения в odbc.ini:
Url = http://login:pass@host:8123/query?database=default&max_result_bytes=4000000&buffer_size=3000000

Табло просто открывает DSN и ничего не показывает. Ни структуры базы, ни ошибок.
Что с этим можно сделать?

**Comment by yurifal** [comment]
А если нажать на значок Search (лупу), 
который расположен возле жирного креста 
в поле ввода имени Таблицы (Enter table name)?

<img width="262" alt="Screenshot 2020-09-28 at 23 25 36" src="https://user-images.githubusercontent.com/2496788/94482494-069b2680-01e2-11eb-804c-ae205fe67ee1.png">

**Comment by sablastik** [comment]
> А если нажать на значок Search (лупу),
> который расположен возле жирного креста
> в поле ввода имени Таблицы (Enter table name)?
> 
> <img alt="Screenshot 2020-09-28 at 23 25 36" width="262" src="https://user-images.githubusercontent.com/2496788/94482494-069b2680-01e2-11eb-804c-ae205fe67ee1.png">

Нет. Совсем пустота. Этих элементов нет. Как и не происходит запросов метаданных на получение схемы

**Comment by traceon** [comment]
Try to make sure the Driver and DSN are configured correctly using command line tool from iODBC.

**Comment by sablastik** [comment]
> Try to make sure the Driver and DSN are configured correctly using command line tool from iODBC.

Can you describe the process? According to https://github.com/ClickHouse/clickhouse-odbc#configuration-iodbc, I've just edited odbc.ini & odbcinst.ini

**Comment by chrsle** [comment]
@traceon would be great to have the process. trying to connect clickhouse to tableau and cannot figure it out

**Comment by sablastik** [comment]
@traceon I've rechecked my odbc.inn & odbcinst.ini connections - I can connect Tableau to clickhouse server 19.7.5.27, but when I try to use DSN with the same parameters changing IP to 20.6.6.7 - nothing happens

**Comment by sablastik** [comment]
I've found that database in DSN to 20.6.6.7 is mandatory. While connection to 19.7.5.27 without pointing out database leads to 'Select Database' dialog:
![image](https://user-images.githubusercontent.com/16276988/94928667-b6b3ac80-04cc-11eb-8a21-e4a02cf9bef0.png)

p.s. however I haven't figured out how to use one line connection specs:
`# Url = https://default:password@localhost:8443/query?database=default&max_result_bytes=4000000&buffer_size=3000000`
it doesn't work neither with 19.7.5.27, nor with 20.6.6.7

**Comment by traceon** [comment]
Tableau Desktop for macOS expects the ODBC driver to be built against iODBC. You are building for iODBC, right?

But first of all, try to make sure `iodbctest` works with the DSN from the console, and after that use Tableau:

```sh
echo "SELECT" |  iodbctest "DSN=MyCliCKHouseDSN"
```

**Comment by chrsle** [comment]
> Tableau Desktop for macOS expects the ODBC driver to be built against iODBC. You are building for iODBC, right?
> 
> But first of all, try to make sure `iodbctest` works with the DSN from the console, and after that use Tableau:
> 
> ```shell
> echo "SELECT 1" | iodbctest "DSN=MyClickHouseDSN"
> ```

Ok. I don't think I'm there yet. How do I create the driver?

**Comment by traceon** [comment]
How do you get `19.7.5.27` working if you don't have a driver?

**Comment by chrsle** [comment]
> How do you get Tableau working with `19.7.5.27` if you don't have a driver?

I don't have that either. I don't have anything.

**Comment by traceon** [comment]
Please take a look at `README.md` it the root of the repo.

**Comment by traceon** [comment]
To use Tableau Desktop with ClickHouse server you'll need:

**Comment by sablastik** [comment]
> Tableau Desktop for macOS expects the ODBC driver to be built against iODBC. You are building for iODBC, right?
> 
> But first of all, try to make sure `iodbctest` works with the DSN from the console, and after that use Tableau:
> 
> ```shell
> echo "SELECT 1" | iodbctest "DSN=MyClickHouseDSN"
> ```

@traceon I've tried to check with this command:
```
echo "SELECT 1" | iodbctest "DSN=ClickNew"

iODBC Demonstration program
This program shows an interactive SQL processor
Driver Manager: 03.52.1319.0723
Driver: 1.1.8.20200710 (CLICKHOUSEODBCW.DLL)

1: SQLExec = Connection refused (1) SQLSTATE=HY000
SQL>
SQL>
Have a nice day.%                  
```

**Comment by slabko** [comment]
Closing do to lack of feedback. Feel free to reopen in case there is some extra information or feedback.
